File: content/code/fahrenheit_to_celsius_docstring.R
#' Convert Fahrenheit to Celsius
#'
#' @param temp_f A numeric vector of temperatures in Fahrenheit.
#' @return A numeric vector of temperatures in Celsius.
fahrenheit_to_celsius <- function(temp_f)
{
  temp_c <- (temp_f - 32.0) * (5.0/9.0)
  temp_c
}

File: content/code/fahrenheit_to_celsius_docstring.py
def fahrenheit_to_celsius(temp_f: float) -> float:
    """
    Converts a temperature in Fahrenheit to Celsius.

    Parameters
    ----------
    temp_f : float
        The temperature in Fahrenheit.

    Returns
    -------
    float
        The temperature in Celsius.
    """

    temp_c = (temp_f - 32.0) * (5.0/9.0)
    return temp_c
